src/Context/FeatureContext.php

```diff
@@ -976,6 +978,7 @@ public function create_config( $subdir = '', $extra_php = false ) {
 		if ( $config_cache_path && file_exists( $config_cache_path ) ) {
 			copy( $config_cache_path, $run_dir . '/wp-config.php' );
 		} else {
+			// TODO: Fail scenario if wp-config.php cannot be written.
```

Comment:
Technically this is already the case, the error message was that `wp-config.php` was not found, but I think it would make sense to throw earlier here already.

Reply:
Opened https://github.com/wp-cli/config-command/pull/166 which would resolve this